microprofile-lra/README.adoc

```diff
@@ -0,0 +1,851 @@
+
+You might remember that we need to propagate the LRA id (LRA context) in the `LRA.LRA_HTTP_CONTEXT_HEADER`. However, if
+we make the outgoing JAX-RS call in the JAX-RS method that already carries an active LRA context, the context is
+automatically added to the outgoing call. So we don't need to pass it manually to each outgoing call.
+
+Now we are ready to propagate LRA started in Participant 1 to the Participant 2, enlist both in the newly started
+LRA, and finish the LRA when the Participant 1 ends its `work` method.
+
+Redeploy the application into the {productName} as showed in <<solution>>. Then you can repeat the calls to the
+`LRAParticipant1` JAX-RS resource as we used them previously:
+
+[source,bash]
+----
+$ curl http://localhost:8080/microprofile-lra/participant1/work
+----
+
+or if you want to simulate LRA failure as:
+
+[source,bash]
+----
+$ curl "http://localhost:8080/microprofile-lra/participant1/work?failLRA=true"
+----
+
+But this time, you will see the LRA is propagated to the `LRAParticipant2` and its (complete or compensate) callbacks
+are invoked by the LRA coordinator in the same way as for `LRAParticipant1`:
+
+[source,bash]
+----
+INFO  [org.wildfly.quickstarts.microprofile.lra.LRAParticipant1] (default task-1) Executing action of Participant 1 enlisted in LRA http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca949a_-4998614b_64e74427_38b that was assigned http://localhost:8080/lra-coordinator/lra-coordinator/recoveryhttp%3A%2F%2Flocalhost%3A8080%2Flra-coordinator%2Flra-coordinator%2F0_ffff0aca949a_-4998614b_64e74427_38b/0_ffff0aca949a_-4998614b_64e74427_38d participant Id.
+
+INFO  [org.wildfly.quickstarts.microprofile.lra.LRAParticipant2] (default task-2) Executing action of Participant 2 enlisted in LRA http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca949a_-4998614b_64e74427_38b that was assigned http://localhost:8080/lra-coordinator/lra-coordinator/recoveryhttp%3A%2F%2Flocalhost%3A8080%2Flra-coordinator%2Flra-coordinator%2F0_ffff0aca949a_-4998614b_64e74427_38b/0_ffff0aca949a_-4998614b_64e74427_38f participant Id.
+
+INFO  [org.wildfly.quickstarts.microprofile.lra.LRAParticipant1] (default task-5) Compensating action for Participant 1 (http://localhost:8080/lra-coordinator/lra-coordinator/recoveryhttp%3A%2F%2Flocalhost%3A8080%2Flra-coordinator%2Flra-coordinator%2F0_ffff0aca949a_-4998614b_64e74427_38b/0_ffff0aca949a_-4998614b_64e74427_38d) in LRA http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca949a_-4998614b_64e74427_38b.
+
+INFO  [org.wildfly.quickstarts.microprofile.lra.LRAParticipant2] (default task-5) Compensating action for Participant 2 (http://localhost:8080/lra-coordinator/lra-coordinator/recoveryhttp%3A%2F%2Flocalhost%3A8080%2Flra-coordinator%2Flra-coordinator%2F0_ffff0aca949a_-4998614b_64e74427_38b/0_ffff0aca949a_-4998614b_64e74427_38f) in LRA http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca949a_-4998614b_64e74427_38b.
+----
+
+//Bootable JAR
```

Comment:
@xstefank @zhantemirov this is optional but I believe a good change wrt user experience and our priorities... I think we should move this bootable jar section include, and the one after wrt to openshift, to https://github.com/wildfly/quickstart/pull/778/files#diff-9473084512ad678e1593a36499d53db5eb7e5725ac6760364e4a139c1e67283aR157 , ie to include those before the create-from-scratch instructions start. This change will put the bootable-jar and openshift at same level as baremetal, right now it kind of gives baremetal the spotlight, which is undesirable considering our priorities.